examples/linearCylinderPlaneStrain/linearCylinderPlaneStrain.m

```diff
@@ -0,0 +1,149 @@
+%md```math 
+%md \mathbf{\mathit{u}}_z(r, \theta, z=0)=0\;\;\quad\forall\; (r,\theta) \\
+%md u_z(r, \theta, z=L_z)=0\;\;\quad\forall\ (r,\theta)
+%md```
+%md
+%md## Analytic solution
+%md
+%md The analytical solution is obtained using the Navier's equation, imposing no temperature variation and no volumetric forces field. Thus the displacements field $\mathbf{\mathit{u}}$ solution satisfies:
+%md```math
+%md  \nabla (\nabla . u(r,\theta,z) )  = 0  
+%md```
+%md Due to the symmetry of the problem $\mathbf{\mathit{u_{\theta}}} = 0 $ and also $\mathbf{ \mathit{ u (r,\theta,z) } } = \mathbf{ \mathit{ u(r,z) } } $. Thus, according to the boundary conditions stated above $\mathit{u_z(r,z)=0}$ and the radial displacements field $\mathit{u_r(r)}$ only varies with $r$. Therafter by imposing the boundary conditions stated above and substituting ($E$, $\nu$) into Lamé parameters ($\lambda=\frac{ E\nu }{(1 + 2\nu )(1 - 2\nu )}$ and $\mu=\frac{ E\nu }{(1 + 2\nu )}$) we obtain:
+%md```math 
+%md u_r(r) = Ar + \dfrac{B}{r} \quad \text{and} \quad u_z(z)  \\
+%md A = \dfrac{(1+\nu)(1-2\nu)R_i^2p_i}{E(R_e^2-R_i^2)}, \quad
+%md B = \dfrac{(1+\nu)R_i^2R_e^2p_i}{E(R_e^2-R_i^2)}
+%md```
+%md### Numerical solution
+%md Before defining the structs, the workspace is cleaned, the ONSAS directory is added to the path and scalar geometry and material parameters are defined:
+clear all, close all
+% add path
+addpath( genpath( [ pwd '/../../src'] ) ) ;
+% scalar parameters
+E = 1e6 ; nu = 0.3 ; p = 1 ; L = .75 ;
+%md
+%md
+%md### MEBI parameters
+%md
+%md#### materials
+%md The constitutive behavior of the material considered is isotropic linear elastic.
+%md Since only one material is considered, the structs defined for the materials contain only one entry:
+materials.hyperElasModel  = 'linearElastic' ;
+materials.hyperElasParams =  [ E nu ]       ;
+%md
+%md#### elements
+%md 
+%md In this model three kinds of elements are used: `triangle` for the solid, `edges` to add external loads and `nodes`. Since three kinds of elements are used, the struct has length 3: 
+elements(1).elemType = 'node';
+elements(2).elemType = 'edge';
+elements(2).elemCrossSecParams = L ;
+elements(3).elemType = 'triangle';
+elements(3).elemTypeParams = 2 ;
+elements(3).elemCrossSecParams = L ;
+%md where `elemCrossSecParams` field sets the thickness of the edge and `elemTypeParams` sets the plane strain triangle element.  
+%md
+%md#### boundaryConds
+%md Three BCs are considered, one corresponding to a load and two for displacements.
+%md The first two BCs constrain displacements in $x$ and $y$ local directions respectively:
+boundaryConds(1).imposDispDofs = [1] ;
+boundaryConds(1).imposDispVals = [0] ;
+boundaryConds(2).imposDispDofs = [3] ;
+boundaryConds(2).imposDispVals = [0] ;
+%md then the third BC introduced in local coordinates sets the radial internal pressure (here is used as a convention that compression to the surface is positive):
```

Comment:
```suggestion
%md then the third BC corresponds to the pressure. It is introduced in `local` coordinates:
```

Reply:
this is something that we should explain better... there is no easy convention on this... it depends on the sense of the mesh done in gmsh...